plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -114,7 +133,17 @@
     // to druid will always have the TABLE_CATALOG set to DRUID_CATALOG
     private static final String DRUID_CATALOG = "druid";
     // All the datasources in Druid are created under schema "druid"
-    public static final String DRUID_SCHEMA = "druid";
+    private static final String DRUID_SCHEMA = "druid";
+    private static final Calendar UTC_CALENDAR = Calendar.getInstance(TimeZone.getTimeZone(UTC));
```

Comment:
Does use of Calendar need to be synchronized?

Reply:
I don't think we need to synchronize it as we get the offset seconds from the Calendar and adjust the Timestamp.